### DISCUSSION TITLE
Really tried to get it to work with Travis-CI

### DISCUSSION THREAD

**Discussion by acgetchell** [pkgxdev/pkgx#849] [Help]
Details:

https://app.travis-ci.com/github/acgetchell/CDT-plusplus/builds/267070588

**Comment by mxcl** [comment]
Error specifically is:

```
nothing provides: eval
```

k so apparently whatever shell Travis CI is running has no `eval`, wtf?

This will require modifying the installer. In the meantime you can just download the pkgx binary per the other install instructions:

```sh
curl -Ssf https://pkgx.sh/$(uname)/$(uname -m).tgz | sudo tar xz -C /usr/local/bin
```

**Reply by mxcl** [reply]
Probs I should just remove the `eval` stuff from the installer, the original thinking was to add shell integration to CI/Docker etc. but generally this doesn't work since all steps are independent shells. The installer should be as simple as possible to avoid issues like this.
